src/screenControl.py

```diff
@@ -514,3 +522,25 @@ def moveCursor(self):
     def getKey(self):
         charCode = self.stdscr.getch()
         return mapping.get(charCode, '')
+
+    def toggleXMode(self):
+        if self.mode != X_MODE:
+            self.mode = X_MODE
+        else:
+            self.mode = SELECT_MODE
+        self.printAll()
+
+    def printXMode(self):
+        if self.mode == X_MODE:
+            (maxy, _) = self.scrollBar.screenControl.getScreenDimensions()
+            topY = maxy - 2
+            minY = self.scrollBar.getMinY() - (1 if not self.scrollBar.activated else 0)
+            for i in range(minY, topY + 1):
+                self.stdscr.addstr(i, 1, lbls[i - minY])
```

Comment:
what do you think about making the X mode selectors only appear next to lines that are actual matches?

that would give us more efficient usage of available characters **and** avoid the awkward situation where you hit a key that doesnt correspond to an actual line match